### ISSUE TITLE
measurement-kit v0.8.1

### ISSUE THREAD

**Issue by bassosimone** [measurement-kit/measurement-kit#1489]
Tracking epic for the last remaining issues before we can bless v0.8.0.

**Comment by bassosimone** [comment]
I renamed the epic as `v0.8.1` because there are bugs in the IM tests that need to be fixed and included into the next release of measurement-kit.

**Comment by bassosimone** [comment]
So, we can now close this epic issue, since all related issues were closed. (It took me some time to realize I left this open, but all pending issues were either closed or needed to be deferred by v0.8.1 ship date.)
